tide_site.module

```diff
@@ -662,3 +662,24 @@ function tide_site_search_api_index_items_alter(IndexInterface $index, array &$i
 function tide_site_tide_api_jsonapi_custom_query_parameters_alter(&$custom_params) {
   $custom_params[] = 'site';
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function tide_site_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $route = \Drupal::routeMatch()->getRouteObject();
```

Comment:
@vincent-gao you can get the view object from form_state, no need to use route match.
```
$view = $form_state->get('view');
```

Reply:
ok. solved. thanks.